.github/workflows/github-cxx-qt-tests.yml

```diff
@@ -376,8 +378,8 @@ jobs:
       if: steps.rust-tools-cache.outputs.cache-hit != 'true'
       # Do not build with storage backends enabled, we only need local
       run: |
-        cargo install --no-default-features sccache
-        cargo install mdbook mdbook-linkcheck
+        cargo install --no-default-features --version 0.7.7 sccache
+        cargo install --version 0.4.36 mdbook mdbook-linkcheck
```

Comment:
`mdbook` and `mdbook-linkcheck` have different versions they need to be on separate lines
```
cargo install --version 0.4.36 mdbook
cargo install --version 0.7.7 mdbook-linkcheck
```